interop.md

```diff
@@ -391,3 +392,37 @@ public class JavaClass {
     }
 }
 ```
+# Lint Checks for Interoperability
+
+## Requirements
+* **Android Studio version required:** 3.2 Canary 10 or later
+* **Lint version required:** 26.2.0-alpha10 or later
+
+## Supported Checks
+There are now Android Lint checks that will help you detect and flag some of the interoperability issues described above. Only issues in Java (for Kotlin consumption) are detected currently. Specifically, the supported checks are:
+* Unknown Nullness
+* Property Access
+* No Hard Kotlin keywords
+* Lambda Parameters Last
+
+## How to use from Android Studio
+To enable these checks, go to **File > Preferences > Editor > Inspections** and check the rules that you want to enable under Kotlin Interoperability:
+
+<img src="{{ site.baseurl }}/assets/kotlin_interop_checks_settings.png"/>
+
+Once you have checked the rules you would like to enable, the new checks will run when you run your code inspections (**Analyze > Inspect Code…**)
+
+## How to use from command-line builds
+To enable these checks from the command-line builds, add the following line in your `build.gradle` file:
+
+```groovy
+android {
+
+    ...
+
+    lintOptions {
+        check 'Interoperability'
+    }
+}```
+
+`./gradlew lint` will now run the Kotlin Interoperability checks.
```

Comment:
Maybe alternatively add something like this:

If you for example want to apply all the interoperability checks *except* the nullness check (which is very verbose), you can use
check 'Interoperability'
disable 'UnknownNullness'